resources/js/Pages/Galleries/hooks/useGalleryDrafts.ts

```diff
@@ -128,7 +139,7 @@ export const useGalleryDrafts = (givenDraftId?: number, disabled?: boolean): Gal
     };
 
     const setDraftTitle = (title: string): void => {
-        setDraft({ ...draft, title });
+        setDraft((previousDraft) => ({ ...previousDraft, title }));
```

Comment:
a sneaky one, should use previous draft over `draft` otherwise unexpected things may happen